### PR TITLE
add message for empty teachers

### PR DESCRIPTION
**Pull request by sergeyevalex90**
Add message if there are no subscriptions to teacher

### CODE REVIEW

File: front-end/src/components/Pricing/Pricing.jsx [hunk 1]
```diff
@@ -38,20 +53,35 @@ const Pricing = () => {
...
-          <PricingCard key={item._id} pricing={item} user={userData} teacher={teacher} />
-        ))}
-      </div>
-    </>
+    fetchSubscriptions(currentUser.roleId);
+  }, [currentUser]);
+
+  return (
+    isAuthenticated &&
+    currentUser.role === 'student' &&
```
Comment (andriydzhrvts) [inline front-end/src/components/Pricing/Pricing.jsx]: Use roles constants

File: front-end/src/components/Pricing/Pricing.jsx [hunk 2]
```diff
@@ -38,20 +53,35 @@ const Pricing = () => {
...
+          {pricing.map((item) => (
+            <PricingCard key={item._id} pricing={item} user={userData} teacher={teacher} />
+          ))}
+        </div>
+      </>
+    )
   );
 
-  return pricingCards;
+  //return pricingCards;
```
Comment (andriydzhrvts) [inline front-end/src/components/Pricing/Pricing.jsx]: Remove comment

File: front-end/src/components/Pricing/Pricing.jsx (hunk 1 above)
Comment (sergeyevalex90) [reply front-end/src/components/Pricing/Pricing.jsx]: Done

File: front-end/src/components/Pricing/Pricing.jsx (hunk 2 above)
Comment (sergeyevalex90) [reply front-end/src/components/Pricing/Pricing.jsx]: Done

File: front-end/src/components/Pricing/Pricing.jsx [hunk 3]
```diff
@@ -38,20 +53,34 @@ const Pricing = () => {
...
-      <TeacherSelect user={currentUser} chooseTeacher={handleTeacher} teacher={teacher} />
-      <div className={styles.pricing}>
-        {pricing.map((item) => (
-          <PricingCard key={item._id} pricing={item} user={userData} teacher={teacher} />
-        ))}
-      </div>
-    </>
-  );
+    fetchSubscriptions(currentUser.roleId);
+  }, [currentUser]);
```
Comment (DeLettenhove) [inline front-end/src/components/Pricing/Pricing.jsx]: Don't use object in dependency array, use primitive values.

File: front-end/src/components/Pricing/Pricing.jsx [hunk 4]
```diff
@@ -38,20 +53,34 @@ const Pricing = () => {
...
+
+  const studentRole = isAuthenticated && currentUser.role === 'student';
```
Comment (DeLettenhove) [inline front-end/src/components/Pricing/Pricing.jsx]: You can get current user only when you authentificated already.

Comment (sergeyevalex90) [reply front-end/src/components/Pricing/Pricing.jsx]: fixed

File: front-end/src/components/Pricing/Pricing.jsx (hunk 3 above)
Comment (sergeyevalex90) [reply front-end/src/components/Pricing/Pricing.jsx]: Fixed

**Review by andriydzhrvts — APPROVED** [review]

**Pull request merged**